src/content/basics/automatic-termination-of-bad-nodes/index.md

```diff
@@ -10,12 +10,14 @@ owner:
 
 # Automatic termination of unhealthy nodes
 
+{{< platform_support_table aws="ga=v12.6.0" azure="ga=v13.1.0" kvm="roadmap=https://github.com/giantswarm/roadmap/issues/176" >}}
+
+## Introduction
+
 Degraded nodes in a Kubernetes cluster should be a rare issue, however when it occurs, it can have severe consequences for the workloads scheduled to the affected nodes. The goal should be to detect bad nodes early and remove them from the cluster, replacing them with healthy ones.
 
 Starting with workload cluster release v12.6.0 for AWS and v13.1.0 for Azure, you now have the option to automate the detection and termination of bad nodes. When enabled, all nodes in your cluster are periodically checked. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
 
-This function is currently available on AWS and Azure only.
-
```

Comment:
Removed phrases like this where I found them, since this information is given by the new table.